src/puptoo/process.py

```diff
@@ -155,8 +155,7 @@ def system_profile(
             profile["sap_sids"] = sorted(list(sids))
             if sap.local_instances:
                 inst = sap.local_instances[0]
-                if sap[inst].number not in ["98", "99"]:
-                    profile["sap_instance_number"] = sap[inst].number
+                profile["sap_instance_number"] = sap[inst].number
```

Comment:
This looks correct, but I don't see any validation to check for 0-99 in insights-core.

I'm not sure if its better to check for that inside of puptoo, or if it's better to put in a bug against insights-core to raise an exception if a bad value would be returned. I think the latter is better but what are yalls thoughts?

Reply:
Honestly I am not sure. I added it as an agenda item to the wednesday meeting.

For now let's just go with this PR as it stands.